### PR TITLE
More docs

### PR DESCRIPTION
**Pull request by bigodines**
also fixes a legacy example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,28 +2,39 @@ rebar_cmd
...
+### Purpose
 
-    $ rebar3 compile
+The goal of this plugin is to allow additional commands to rebar3 so that one can use it to manage everything in an erlang project.
+
+Wether it is bringing docker containers up, tagging a new release on git or deleting log files, we should be able to use the same build tool for all of that.
+
+### How it works
+
+This is a very simple and straight forward plugin. Simply tell rebar a list of commands to execute (just like linux aliases) and access them via `rebar3 cmd <you command>`
```
Comment (gootik) [inline README.md]: `your command`

File: README.md [hunk 2]
```diff
@@ -2,28 +2,39 @@ rebar_cmd
...
+
+
+### Usage
 
-Use
----
 
 Add the plugin to your rebar config:
```
Comment (gootik) [inline README.md]: maybe add `(rebar.config)`

File: README.md [hunk 3]
```diff
@@ -2,28 +2,39 @@ rebar_cmd
...
+In this case, we've added a command to bring our docker containers up, and another to sync the git repository with the remote master.
+
+Then you just have to call your plugin directly in an existing application:
 
 
     $ rebar3 cmd docker_up
     ===> Fetching rebar_cmd
     ===> Compiling rebar_cmd
-    <Plugin Output>
+    <Command Output will be here>
```
Comment (gootik) [inline README.md:38]: maybe put actual output from the examples above

File: README.md [hunk 4]
```diff
@@ -2,28 +2,39 @@ rebar_cmd
...
+### Purpose
 
-    $ rebar3 compile
+The goal of this plugin is to allow additional commands to rebar3 so that one can use it to manage everything in an erlang project.
+
+Wether it is bringing docker containers up, tagging a new release on git or deleting log files, we should be able to use the same build tool for all of that.
+
+### How it works
+
+This is a very simple and straight forward plugin. Simply tell rebar a list of commands to execute (just like linux aliases) and access rhem via `rebar3 cmd <your command>`
```
Comment (gootik) [inline README.md]: `access them` not `rhem`

File: README.md (hunk 3 above)
Comment (bigodines) [inline README.md:38]: meh

**Pull request merged**